dataenum-processor/src/main/java/com/spotify/dataenum/processor/generator/value/ValueTypeFactory.java

```diff
@@ -267,11 +267,17 @@ private static MethodSpec createToString(OutputValue value) {
     for (Parameter parameter : value.parameters()) {
       String fieldName = parameter.name();
 
+      String valueFormat = parameter.redacted() ? "\"***\"" : "$1L";
+
       if (first) {
         first = false;
-        result.addStatement("builder.append(\"$2L{$1N=\").append($1L)", fieldName, value.name());
+        result.addStatement(
+            String.format("builder.append(\"$2L{$1N=\").append(%s)", valueFormat),
+            fieldName,
+            value.name());
       } else {
-        result.addStatement("builder.append(\", $1N=\").append($1L)", fieldName);
```

Comment:
Could we maybe have a test in the dataenum-processor/src/test suite for this? Just so there is an example of the what the generated code looks like.

Reply:
I would rather not - because the point of this change isn't what the code looks like, but how it behaves. So it's better to test it through using it. Verifying how redaction is implemented is the kind of thing that'll mostly make it harder to change, I think. Make sense?

Reply:
(As a side note, I guess it's because of testing from the `integration-test` package that codecov thinks the 'redacted' branch isn't covered. It is, but later...)

Reply:
Well yes, but at the same time that would mean that this is the first feature of the dataenum-processor that isn't covered in those tests, and I think the harder-to-change part of having that entire test suite is something we're already aware of. Does this mean we should move towards removing that entire test suite, or accept that there are things not covered by it? I think there is some value in terms of documentation, that you can easily get an idea of what code is generated, even though it makes changes harder.